modules/RTC/TraceablePeerConnection.js

```diff
@@ -371,8 +371,7 @@ TraceablePeerConnection.prototype.isSimulcastOn = function() {
         // doesn't have a reference to the config options, so we have
         // to do it here
         && (!RTCBrowserType.isFirefox()
-            || this.options.enableFirefoxSimulcast)
-        && !this.isP2P;
+            || this.options.enableFirefoxSimulcast);
```

Comment:
You're right, we should not be doing simulcast in p2p. I moved the check in JingleSessionPC (in this commit).